### PR TITLE
Adding Spark example to Write Pojo classes to Stream 

### PR DESCRIPTION
**Pull request by kishorebhat-p**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

Added the class SensorReading and class ProducerSparkWithPojo  to showcase writing Pojo as Json Object to Pulsar Queue 

### Modifications

Based on the documentation written example class to write Pojos to Pulsar 
This mightbe more real life examples for application than bytes stream 

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docker-compose/standalone-dashboard/docker-compose.yml [hunk 1]
```diff
@@ -23,9 +23,9 @@ services:
 
   standalone:
     image: apachepulsar/pulsar
-    expose:
-      - 8080
-      - 6650 
+    ports:
```
Comment (sijie) [inline docker-compose/standalone-dashboard/docker-compose.yml]: can you separate this change into a separated pull request?

File: examples/spark/src/main/java/org/apache/spark/streaming/receiver/example/ProducerSparkWithPojo.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package org.apache.spark.streaming.receiver.example;
```
Comment (sijie) [inline examples/spark/src/main/java/org/apache/spark/streaming/receiver/example/ProducerSparkWithPojo.java]: we need to add license header to every file. you can run `mvn license:format` to do so.

Comment (kishorebhat-p) [reply examples/spark/src/main/java/org/apache/spark/streaming/receiver/example/ProducerSparkWithPojo.java]: Ok  will do 
Thanks 
Kishore

File: docker-compose/standalone-dashboard/docker-compose.yml (hunk 1 above)
Comment (kishorebhat-p) [reply docker-compose/standalone-dashboard/docker-compose.yml]: Ok  will do 
Thanks 
Kishore

File: examples/spark/src/main/java/org/apache/spark/streaming/receiver/example/ProducerSparkWithPojo.java (hunk 2 above)
Comment (kishorebhat-p) [reply examples/spark/src/main/java/org/apache/spark/streaming/receiver/example/ProducerSparkWithPojo.java]: Made changes to add License details in New files as mentioned using mvn:license:format

**Comment by jiazhai** [comment]
run integration tests

**Comment by sijie** [comment]
run integration tests

**Pull request merged**